cider/exec/nextgen/jitlib/llvmjit/LLVMJITFunction.cpp

```diff
@@ -183,6 +184,14 @@ JITValuePointer LLVMJITFunction::getArgument(size_t index) {
                                             false);
   }
 }
+
+IfBuilderPointer LLVMJITFunction::getIfBuilder() {
+  return std::make_unique<LLVMIfBuilder>(func_, *ir_builder_);
+}
```

Comment:
`createIfBuilder` may be better than `getIfBuilder`, cause we always create a new instance.